src/utils/__tests__/create-utils.spec.ts

```diff
@@ -58,6 +59,9 @@ const mockComponentData = {
   metadata: {
     name: 'test-component',
     namespace: 'test-ns',
+    annotations: {
+      'skip-inital-checks': 'true',
```

Comment:
```suggestion
      'skip-initial-checks': 'true',
```

Reply:
Bad copy paste job from the story. Updated.